pages/index.jsx

```diff
@@ -1,14 +1,33 @@
+import MovieList from '@components/movie/MovieList'
 import Head from 'next/head'
 
-const Home = () => {
+const Home = ({ data }) => {
+  if (data.success === false)
```

Comment:
No need to do this. Please optimize.

Reply:
I have no other ideas than this. Should it be completely removed? 
Maybe we can revisit the error handling part later after implementing the happy cases first.